### PR TITLE
changes required to upgrade to bootstrap 4. 

### PR DESCRIPTION
**Pull request by kghmanuel**
pagination heavily affected due to changes from bootstrap 3. sample shots:

image

image

image

### CODE REVIEW

**Comment by mariannemyers** [comment]
Since you've switched to using "Bootstrap Blue" (as I call it), we should also change the selected filter to be btn-primary instead of btn-success:

https://github.com/marklogic-community/grove-react-ui/blob/b0e7f8f153145f8a55ee06f7adc9099bb253153f/src/components/views/search/Facets/CurrentFilters.js#L31

I think this doesn't look like it fits with the use of the dodgerblue color:

image

**Comment by mariannemyers** [comment]
I see that Glyphicons were removed in Bootstrap v4, so you changed this project to use Font Awesome instead.  A package addition requires a FOSS review.  I'll take care of that step.

**Comment by kghmanuel** [comment]
> we should also change the selected filter to be btn-primary instead of btn-success

Done. It wasn't intentional to switch the colors over. :) more of i didn't want to introduce too many entries in index.css.

File: package.json [hunk 1]
```diff
@@ -11,26 +11,27 @@
...
-    "react-router-dom": "^4.2.2",
+    "lodash": "^4.17.15",
+    "query-string": "^6.13.0",
+    "react": "^16.13.1",
+    "react-bootstrap": "^1.0.1",
+    "react-codemirror2": "^7.2.1",
+    "react-dom": "^16.13.1",
+    "react-json-tree": "^0.11.2",
+    "react-redux": "^7.2.0",
+    "react-router-bootstrap": "^0.25.0",
```
Comment (mariannemyers) [inline package.json]: We believe this is no longer used with your refactor.  Can you also remove it from dependencies?  Dave Grant caught this during a team review of this PR today.

File: src/components/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  for (var i = min; i < max; ++i) {
+    if (i === 1 || i === items) {
+      continue;
+    }
+    pageNumbers.push(
+      <BSPagination.Item active={i === activePage}>{i}</BSPagination.Item>
+    );
+  }
+  if (items - max > 0) {
+    pageNumbers.push(<BSPagination.Ellipsis />);
```
Comment (mariannemyers) [inline src/components/Pagination.js]: After a hard refresh (cmd-shift-r) of the page (or first load), clicking on the ellipsis in the UI for the sample person data yields an error involving the Redux layer.  It does not seem to happen on subsequent navigation back to the search results.

image

**Review by mariannemyers — COMMENTED** [review]
Looks good!  Please address the two in-line comments and we'll merge after the FOSS review is complete.

**Pull request merged**